crates/ruff_cli/src/commands/rule.rs

```diff
@@ -10,9 +10,13 @@ use crate::args::HelpFormat;
 
 #[derive(Serialize)]
 struct Explanation<'a> {
+    name: &'a str,
     code: &'a str,
     linter: &'a str,
     summary: &'a str,
```

Comment:
Should summary be deprecated in favor of message_formats?

Reply:
Probably... I suppose we can leave it for now, I don't know if any clients depend on this.